charts/gardener/charts/application/templates/rbac-system.yaml

```
@@ -87,3 +87,25 @@ subjects:
   name: "{{ required ".Values.global.controller.serviceAccountName is required" .Values.global.controller.serviceAccountName }}"
   namespace: garden
 {{- end }}
+
+{{- if .Values.global.scheduler.enabled }}
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRoleBinding
+metadata:
+  name:  garden.sapcloud.io:scheduler
```

```suggestion
  name:  gardener.cloud:scheduler
```

We plan to switch to `gardener.cloud` (please do this for all new scheduler roles/occurrences)